### ISSUE TITLE
Code editor 2: Putting `!` or `not` in front of a function makes assets not appear in autocomplete for any arguments

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#6933]
### Description

If you type 
```
if instance_exists()
``` 
Object assets will appear in autocomplete for the argument. However, if you do this
```
if !instance_exists()
```
Objects will NOT appear in autocomplete. 
This seems to apply to all functions, regardless of what kind of asset they take as an argument. Also applies to both built in functions and user defined functions.

https://github.com/user-attachments/assets/8ff975b5-0489-4ecf-bc7e-7329d681a863

### Expected Change

Autocomplete to work the same regardless of it there's a ! in front of the function or not

### Steps To Reproduce

Type
```
if instance_exists()
``` 
and look at autocomplete. Then type 
```
if !instance_exists()
```
and compare. Can use any function that takes an asset for any of it's arguments.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.800 (Betas)

### Which operating system(s) are you seeing the problem on?

Windows 11

### Are you running GameMaker from inside your Steam library?

None

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by zreedy** [comment]
Fixed

**Comment by caitlinrmcintyre** [comment]
Verified fix in IDE v2024.1000.0.622  Runtime v2024.1000.0.648, thanks.
